doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,141 @@
+.. _recipe_kcs:
+
+KNMI Climate Scenarios
```

Comment:
Add year 2014 to the title. The method in 2006 was different, and likely adaptations will be made in 2023.